diff --git a/pyproject.toml b/pyproject.toml
index 1747d21..216d012 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -12,19 +12,19 @@ include = ["CHANGELOG.md"]
 python = "^3.8.0"
 thrift = ">=0.16.0,<0.21.0"
 pandas = [
-    { version = ">=1.2.5,<2.3.0", python = ">=3.8" }
+    { version = ">=1.2.5", python = ">=3.8" }
 ]
-pyarrow = ">=14.0.1,<17"
+pyarrow = ">=14.0.1"
 
 lz4 = "^4.0.2"
 requests = "^2.18.1"
 oauthlib = "^3.1.0"
 numpy = [
-    { version = "^1.16.6", python = ">=3.8,<3.11" },
-    { version = "^1.23.4", python = ">=3.11" },
+    { version = ">=1.16.6", python = ">=3.8,<3.11" },
+    { version = ">=1.23.4", python = ">=3.11" },
 ]
 sqlalchemy = { version = ">=2.0.21", optional = true }
-openpyxl = "^3.0.10"
+openpyxl = ">=3.0.10"
 alembic = { version = "^1.0.11", optional = true }
 urllib3 = ">=1.26"
 
